conda_forge_tick/audit.py

```diff
@@ -50,14 +50,18 @@
     *[set(stdlib_list(k)) for k in ["2.7", "3.5", "3.6", "3.7"]]
 )
 
-STATIC_EXCLUDES = {
-    "python",
-    "setuptools",
-    "pip",
-    "versioneer",
-    # not a real dep
-    "cross-python",
-} | BUILTINS | set(COMPILER_STUBS_WITH_STRONG_EXPORTS)
+STATIC_EXCLUDES = (
+    {
+        "python",
+        "setuptools",
+        "pip",
+        "versioneer",
+        # not a real dep
+        "cross-python",
+    }
+    | BUILTINS
+    | set(COMPILER_STUBS_WITH_STRONG_EXPORTS)
```

Comment:
This is fine, but do be aware that this is not the full list of possible stubs. Any compiler that only statically links is not in this list. I think this code is fine. This comment is an FYI.

Reply:
Sure, and we should document all the stubs somewhere, but something tells me that the actual answer is hidden somewhere in conda-build or pinnings, and I didn't have my spelunking hat on :construction_worker_man:

Reply:
Yep. The answer is in pinnings and changes as that file changes. So much fun!